indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -232,6 +232,46 @@
         </div>
 {% endmacro %}
 
+{% macro _render_judgment_action(abstract) %}
+    {% set action = abstract.state.name %}
+    {% if abstract.can_judge(session.user) %}
+        {% set judge_name = abstract.judge.name %}
+        {% if action == 'withdrawn' %}
+            {% trans name=judge_name -%}
+                <strong>{{ name }}</strong> withdrew this abstract
```

Comment:
Well, if it's been withdrawn, I agree the submitter him/herself should be able to see by whom. After all it's an action on his/her side.